v7/eval.go

```diff
@@ -189,19 +186,11 @@ func entryEvaluator(key string, node *entry, tinfo *userTypeInfo) entryEvalFunc
 			for _, rule := range node.PercentageRules {
 				bucket += rule.Percentage
 				if scaled < bucket {
-					result := rule.Value
-					if logger.enabled(LogLevelInfo) {
-						logger.Infof("Evaluating %% options. Returning %v", result)
```

Comment:
I think this log entry was missed, could you please put it back? Otherwise, it all looks good.

Reply:
We're still logging an entry (it's now done universally for all entries inside `snapshot.valueAndVariationID`).

However you're right in that the log entry has changed: it is now `Returning xxx`, not `Evaluating % options. Returning xxx`.

I made this change because it means that we don't have to have the `Returning %v.` logging logic in three separate places (it's just in the one place inside `Snapshot.valueAndVariationID`) and I thought perhaps the `Evaluating % options` part of the log line wasn't that informative (and slightly misleading, as it's printed _after_ all the % options have been evaluated).

Happy to go either way here.

Reply:
I just added https://github.com/configcat/go-sdk/pull/42 as a prequel to this branch, so that we can easily see what's being changed here and review accordingly.

Reply:
Yep, I already merged it, could you please update this with the new master?

Reply:
@z4kn4fein Done! Sorry about the long delay - I'd totally forgotten this was still hanging around :)